src/common/SPELLS/BFA/AzeriteTraits/Paladin.js

```diff
@@ -22,11 +22,6 @@ export default {
     name: 'Grace of the Justicar',
     icon: 'spell_holy_healingaura',
   },
-  CONCENTRATED_MENDING: {
-    id: 272260,
-    name: 'Concentrated Mending',
-    icon: 'inv_offhand_1h_pvpdraenors1_d_02',
-  },
 
```

Comment:
Since you removed this and renamed it to ``CONCENTRATED_MENDING_HEALING`` you should go and find it's usages in the paladin traits and change the names there, so as to not break anything.

Reply:
Okay, fixed that. Only seems to be referenced in one place in the paladin parser.

Reply:
There is another reference in `Parser/Core/Modlues/Features/SpellInfo.js` for stat scaling of the healing spell

Reply:
Jep, those were all occurrences of `CONCENTRATED_MENDING`